### DISCUSSION TITLE
Is there a way to use headless mode in Cypress while Synpress is working as a plugin of Cypress?

### DISCUSSION THREAD

**Discussion by emmabubu** [Synthetixio/synpress#688] [Q&A]
@drptbl 
Thanks for providing excellent usage examples for Synpress at https://github.com/drptbl/synpress-examples/. They are very helpful.
I've tried different examples for the three frameworks and noticed that Synpress  and Playwright both provide commands to run in headless mode, but Cypress doesn't.
I'm writing test automation code for my company's products(several dapps) and I'm using Cypress as framework. Right now it can only run in headed mode. It will very helpful to integrate it into CI if can run in headless mode.

Thanks,
Emma

**Comment by sebastiendan** [comment]
Not really an answer for you but JFYI I've been able to run synpress with cypress in the CI in headful mode (headless isn't a requirement for running in the CI).

**Reply by emmabubu** [reply]
@sebastiendan, thank you for your helping. I will try to  run synpress with cypress in headful mode in github actions
